examples/mediterranean_simulation_with_ecco_restoring.jl

```diff
@@ -26,35 +26,38 @@ using Dates
 
 # ## Grid Configuration for the Mediterranean Sea
 #
-# The script defines a high-resolution grid to represent the Mediterranean Sea, specifying the domain in terms of longitude (λ₁, λ₂),
-# latitude (φ₁, φ₂), and a stretched vertical grid to capture the depth variation (`z_faces`).
-# The grid resolution is set to approximately 1/15th of a degree, which translates to a spatial resolution of about 7 km.
-# This section demonstrates the use of the LatitudeLongitudeGrid function to create a grid that matches the
+# The script defines a high-resolution grid to represent the Mediterranean Sea, specifying the domain
+# in terms of longitude (λ₁, λ₂), latitude (φ₁, φ₂), and a stretched vertical grid (via `StretchedInterfaces`)
+# to capture the depth variation.
+# The grid resolution is set to approximately 1/15th of a degree, which corresponds to about 7 km.
+#
+# This section demonstrates using the `LatitudeLongitudeGrid` to create a grid that matches the
 # Mediterranean's geographical and bathymetric features.
 
 λ₁, λ₂  = ( 0, 42) # domain in longitude
 φ₁, φ₂  = (30, 45) # domain in latitude
-z_faces = stretched_vertical_faces(depth = 5000,
-                                   surface_layer_Δz = 2.5,
-                                   stretching = PowerLawStretching(1.070),
-                                   surface_layer_height = 50)
+
+z = StretchedInterfaces(; depth = 5000,
+                        surface_layer_Δz = 2.5,
+                        stretching = PowerLawStretching(1.07),
+                        surface_layer_height = 50)
 
 Nx = 15 * Int(λ₂ - λ₁) # 1/15 th of a degree resolution
 Ny = 15 * Int(φ₂ - φ₁) # 1/15 th of a degree resolution
-Nz = length(z_faces) - 1
+Nz = length(z)
```

Comment:
Shouldn't this still be `Nz = length(z) - 1` if `z` represents the interfaces?

Reply:
Perhaps we can think of `z` as representing the coordinate more generally but it is called "Interfaces" now

Reply:
Is `ExponentialCoordinate` more appropriate you reckon?